### PR TITLE
docs(sample): add missing samples

### PR DESCRIPTION
**Pull request by laljikanjareeya**
PR contains following samples:

- storage_object_csek_to_cmek
- storage_compose_file
- storage_download_public_file
- storage_get_service_account
- storage_set_bucket_public_iam
- storage_bucket_delete_default_kms_key
- storage_define_bucket_website_configuration

### CODE REVIEW

File: samples/composeFile.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  const {Storage} = require('@google-cloud/storage');
+
+  // Creates a client
+  const storage = new Storage();
+
+  async function composeFile() {
+    const bucket = storage.bucket(bucketName);
+    const sources = [bucket.file(firstFileName), bucket.file(secondFileName)];
+
+    const destination = bucket.file(destinationFileName);
```
Comment (AVaksman) [inline samples/composeFile.js]: This piece of code only creates a reference to (as of this moment) a non existent file object. It doesn't seem to be used anywhere in the sample. Is it needed?

File: samples/composeFile.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    await bucket.combine(sources, destination);
```
Comment (AVaksman) [inline samples/composeFile.js]: Can `sources` and `destination` all be just strings?

File: samples/downloadPublicFile.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  const http = require('http');
+  const fs = require('fs');
+
+  function downloadPublicFile() {
+    const file = fs.createWriteStream(destFileName);
+
+    http.get(
+      `http://storage.googleapis.com/${bucketName}/${srcFileName}`,
+      response => {
+        response.pipe(file);
```
Comment (AVaksman) [inline samples/downloadPublicFile.js:47]: is `response` a stream? [docs](https://nodejs.org/api/http.html#http_http_get_url_options_callback)
Isn't argument for `pipe()` needs to be a writable stream?

File: samples/downloadPublicFile.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  // const srcFileName = 'Remote file to download, e.g. file.txt';
+  // const destFileName = 'Local destination for file, e.g. ./local/path/to/file.txt';
+
+  const http = require('http');
+  const fs = require('fs');
+
+  function downloadPublicFile() {
+    const file = fs.createWriteStream(destFileName);
+
+    http.get(
```
Comment (AVaksman) [inline samples/downloadPublicFile.js:44]: Why not `file.download()`?
Here are some existing [samples](https://cloud.google.com/storage/docs/access-public-data#storage-download-public-object-java)

File: samples/makeBucketPublic.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  async function makeBucketPublic() {
+    const bucket = storage.bucket(bucketName);
+
+    const [policy] = await bucket.iam.getPolicy();
+    policy.bindings.push({
+      role: 'roles/storage.objectViewer',
+      members: ['allUsers'],
+    });
+
+    await bucket.iam.setPolicy(policy);
```
Comment (AVaksman) [inline samples/makeBucketPublic.js]: Why not [`bucket.makePublic()`](https://github.com/googleapis/nodejs-storage/blob/master/src/bucket.ts#L2847)?

**Review by AVaksman — CHANGES_REQUESTED** [review]

File: samples/downloadPublicFile.js (hunk 4 above)
Comment (laljikanjareeya) [reply samples/downloadPublicFile.js:44]: `file.download`  will go through the authentication process, and Here File is public So I don't think we need to storage class any more.

File: samples/downloadPublicFile.js (hunk 3 above)
Comment (laljikanjareeya) [reply samples/downloadPublicFile.js:47]: Yes, `file` is writable stream.

**Pull request closed without merging**